### ISSUE TITLE
TouchableOpacity does not receive press event on real device

### ISSUE THREAD

**Issue by vinhtq** [bolan9999/react-native-spring-scrollview#13]
```
return (
     <SafeAreaView style={{ flex: 1 }}>
          <SpringScrollView>
            {arr.map((i, index) =>
                <TouchableOpacity
                    key={index}
                    onPress={() => console.log('clicked')}
                >
                    <Text>
                     Modify the '_contentCount','_bounces' and '_scrollEnabled' in
                     BouncesExample.js to check if VerticalScrollView works well.
                     </Text>
                 </TouchableOpacity>
            )}
          </SpringScrollView>
     </SafeAreaView>
 )
```

On Android 7.0 real device, the press event of TouchOpacity cannot be fired

I even tried the Examples/ComplexExample.js but it also does not work

**Comment by bolan9999** [comment]
It is not easy to fire , I have already reproduced this issue. And I will fix it soon.

**Comment by vinhtq** [comment]
I took me days to find the solution for this issue. Instead of using `TouchableOpacity` from `react-native`, I used from 'react-native-gesture-handler' library. Now it works perfectly

**Comment by bolan9999** [comment]
fixed on Version 2.0.3
